ts/src/bitmex.ts

```diff
@@ -261,6 +263,174 @@ export default class bitmex extends Exchange {
+                const isDepositEnabled = this.safeValue (chain, 'depositEnabled', false);
+                const isWithdrawEnabled = this.safeValue (chain, 'withdrawalEnabled', false);
+                const active = (isDepositEnabled && isWithdrawEnabled);
+                if (isDepositEnabled) {
+                    depositEnabled = true;
+                }
+                if (isWithdrawEnabled) {
+                    withdrawEnabled = true;
+                }
+                networks[network] = {
+                    'info': chain,
+                    'id': networkId,
+                    'network': network,
+                    'active': active,
+                    'deposit': isDepositEnabled,
+                    'withdraw': isWithdrawEnabled,
+                    'fee': withdrawalFee,
+                    'precision': undefined,
+                    'limits': {
+                        'withdraw': {
+                            'min': undefined,
+                            'max': undefined,
+                        },
+                        'deposit': {
+                            'min': undefined,
+                            'max': undefined,
+                        },
+                    },
+                };
+            }
+            const currencyEnabled = this.safeValue (currency, 'enabled');
+            const currencyActive = currencyEnabled || (depositEnabled || withdrawEnabled);
+            const minWithdrawalString = this.safeString (currency, 'minWithdrawalAmount');
+            const minWithdrawal = this.parseNumber (Precise.stringMul (minWithdrawalString, precisionString));
+            const maxWithdrawalString = this.safeString (currency, 'maxWithdrawalAmount');
+            const maxWithdrawal = this.parseNumber (Precise.stringMul (maxWithdrawalString, precisionString));
+            const minDepositString = this.safeString (currency, 'minDepositAmount');
+            const minDeposit = this.parseNumber (Precise.stringMul (minDepositString, precisionString));
+            result[code] = {
+                'id': id,
+                'code': code,
+                'info': currency,
+                'name': name,
+                'active': currencyActive,
+                'deposit': depositEnabled,
+                'withdraw': withdrawEnabled,
+                'fee': undefined,
+                'precision': precision,
+                'limits': {
+                    'amount': {
+                        'min': undefined,
+                        'max': undefined,
+                    },
+                    'withdraw': {
+                        'min': minWithdrawal,
+                        'max': maxWithdrawal,
+                    },
+                    'deposit': {
+                        'min': minDeposit,
+                        'max': undefined,
+                    },
+                },
+                'networks': networks,
+            };
+        }
+        return result;
```

Comment:
hum weird, it seems `fetchCurrencies` had a side-effect 
![image](https://github.com/ccxt/ccxt/assets/43336371/9ea3d704-46e5-4103-bcea-416bcb202710)

Reply:
It shouldnt be a problem of fetchCurrencies, as `.EVOL7D` would be a returned market from `fetchMarkets`. As I remind, I've seen a comment somewhere, where it checked `if (market['active'])` probably for that specific reason, as inactive symbols does not have a ticker available. So, maybe we:
A) adjust tests to avoid fetching orderbook, ticker(s) or other realtime data for inactive symbols
B) adjust bitmex, so the temporarily inactive markets aren't included (not ideal)

Reply:
@ttodua but why we don't get the same error with the master version?

Reply:
on first place, why it did fetchTicker for that symbol? O_o
I'll look into that.

Reply:
but it says "exception from **fetchTicker (BTC/USDT:USDT)**" (no plural **fetchTickers**).
oky give me some time to look into this